data/json/mapgen/nested/city_npc_nested_spawns.json

```diff
@@ -45,18 +45,18 @@
     "object": {
       "mapgensize": [ 10, 10 ],
       "rows": [
-        "__._______",
-        "_L.7______",
-        "_L.7______",
-        "_L.7______",
-        "_L.7______",
-        "_L.7______",
-        "_L.7______",
-        "_L..mtb.__",
-        "_L.....___",
-        "_LLLLLL.__"
+        "  .       ",
+        " L.7      ",
+        " L.7      ",
+        " L.7      ",
+        " L.7      ",
+        " L.7      ",
+        " L.7      ",
+        " L..mtb.  ",
+        " L.....   ",
+        " LLLLLL.  "
       ],
-      "terrain": { "_": "t_null", "-": "t_wall", ".": "t_floor" },
+      "terrain": { "-": "t_wall", ".": "t_floor" },
```

Comment:
same here, `_` changed for `-`

Reply:
No, I don't think so?  It just deleted `"_": "t_null", `.